x-pack/platform/plugins/shared/lens/public/app_plugin/get_application_user_messages.tsx

```diff
@@ -141,7 +141,7 @@ function getMissingIndexPatternsErrors(
   // Check for access to both Management app && specific indexPattern section
   const { management: isManagementEnabled } = core.application.capabilities.navLinks;
   const isIndexPatternManagementEnabled =
-    core.application.capabilities.management.kibana.indexPatterns;
+    core.application.capabilities.management?.kibana?.indexPatterns;
```

Comment:
why TS haven't thrown an error here?Is because management is typed as any or something similar?

Reply:
I had to relax the checks in here due to test mocks which are not complete. In term of regular Kibana runtime it should always be defined.